sdk/management/pipelines/release.yml

```diff
@@ -62,7 +62,7 @@ steps:
   condition: ne(variables['Snapshot'], true)
   inputs:
     mavenPomFile: 'sdk/management/pom.xml'
-    mavenOptions: '-Xmx3072m -Dversion=$(VERSION) -DdevVersion=$(DEVVERSION)-SNAPSHOT'
+    options: '-Dversion=$(VERSION) -DdevVersion=$(DEVVERSION)-SNAPSHOT'
```

Comment:
Why 	Azure-Fluent   change this?

Reply:
Actually, It is my change due to the previous pipeline error. Which was carried by fluent.

Reply:
I'm still testing since I don't know why the pipeline is failed.